website/docs/r/installed_collector.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_installed_collector"
+description: |-
+  Provides a Sumologic (Installed) Collector.
+---
+
+# sumologic_collector
+Provides a [Sumologic (Installed) Collector][1].
+**NOTE**: This resource can only be imported and managed by terraform. Creation of this resource via terrform is not supported.
+
+## Example Usage
+```hcl
+resource "sumologic_installed_collector" "installed_collector" {
+  name        = "vsinghal-mac"
+  category = "macos/vsinghal"
+  ephemeral = true
+  fields =  {
+      _budget =  "Dev_20GB"
```

Comment:
`_budget` is v1 budget which is not recommended for customers anymore, use a different example?